src/player.rs

```diff
@@ -216,8 +220,8 @@ fn rise(
 
 	// Update the player's vertical position
 	match player.translation {
-		Some(vec) => player.translation = Some(Vec2::new(vec.x, new_height)),
-		None => player.translation = Some(Vec2::new(0.0, new_height)),
+		Some(vec) => player.translation = Some(Vec2::new(vec.x, new_height * 8.0)),
+		None => player.translation = Some(Vec2::new(0.0, new_height * 8.0)),
```

Comment:
This caused the cat to jump massively in the air... doesn't look like at all what you had on your screen?